lib/util/errors.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const {
+  Error,
+  SafeMap,
+} = primordials;
+
+const {
+  makeErrorWithCode: _makeErrorWithCode,
+  AbortError,
+  codes: {
+    ERR_INVALID_ARG_TYPE,
+    ERR_INVALID_ERROR_MESSAGE,
+    ERR_MISSING_ERROR_MESSAGE,
+  },
+} = require('internal/errors');
+
+const {
+  DOMException
+} = internalBinding('messaging');
```

Comment:
```suggestion
const {
  DOMException,
} = internalBinding('messaging');
```
or
```suggestion
const { DOMException } = internalBinding('messaging');
```

Reply:
I prefer to keep it across multiple lines